shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -692,6 +692,8 @@ - (BOOL)accessibilityActivate {
     return NO;
   }
   if (!self.node.HasAction(flutter::SemanticsAction::kTap)) {
+    // Prevent Slider to receive a regular tap which will change the value.
+    // This is needed because Slider does not have a semantics tap.
```

Comment:
```suggestion
    // Prevent Slider to receive a regular tap which will change the value.
    // 
    // This is needed because it cause slider to select to middle if it
    // doesn't have a semantics tap. 
```